pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -200,14 +200,7 @@ func (r *Reconciler) handleError(err error, th *v1alpha1.TektonHub) error {
 }
 
 func (r *Reconciler) manageUiComponent(ctx context.Context, th *v1alpha1.TektonHub, hubDir, version string) error {
-	if err := r.validateUiConfigMap(ctx, th); err != nil {
-		th.Status.MarkUiDependencyMissing(fmt.Sprintf("UI config map not present: %v", err.Error()))
-		return v1alpha1.REQUEUE_EVENT_AFTER
-	}
-
-	th.Status.MarkUiDependenciesInstalled()
```

Comment:
I think we need to mark the status of dependencies as true to make tekton hub status as true

Reply:
there is no dependency as such, i removed the condition